src/hooks/DummyHttp.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import mockMovieData from '../mockData/mockMovieData';
+
+/*
+For now it works with mock data (without actual http request)
+*/
+export const useDummyHttp = (dependency) => {
```

Comment:
file nane should be also be useDummyHttp

Reply:
done